dependencies.yaml

```diff
@@ -397,7 +397,7 @@ dependencies:
               - *libcusparse114
   cuda_wheels:
     specific:
-      - output_types: pyproject
+      - output_types: [requirements, pyproject]
```

Comment:
This should be fine, for the reasons mentioned in https://github.com/rapidsai/raft/pull/2563/files#r1938056656